### DISCUSSION TITLE
Some em7455 questions

### DISCUSSION THREAD

**Discussion by macaquinyho** [danielewood/sierra-wireless-modems#44] [General] [answered]
Hi, so i'm planning to get a U7621 router and use a cheap em7455 with a minipcie to m2 adapter inside. Can I flash the module directly from the router and is there any different between usb 2.0 and 3.0 modes and speed?

**Comment by danielewood** [comment, marked as answer]
1. You cant flash directly on the router as the flashing tools are not available compiled for MIPSle32 CPU.
2. In that router, no, since it lacks USB3.0 to the MiniPCIe slot anyways. Even so, USB2 can exceed 260Mbps with these modems, so that is not going to be your bottleneck.

**Comment by macaquinyho** [comment]
Ok thank you, is there anything out of order on [this](https://es.aliexpress.com/item/32971278547.html?)  card? 
Also, I want to flash my to the lastest firmware but I don't need to change my usb ids, how do I not change the usb id with your script?
